PureCN_curation.ipynb

```diff
@@ -0,0 +1,203 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4bb6282",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wm = dm.WorkspaceManager(WORKSPACE)\n",
+    "samples_df = wm.get_samples()\n",
+    "#samples_df = pd.read_csv(\"~/Downloads/sample.tsv\",sep = \"\\t\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "497bdd63",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "samples_df['PureCN_gof'] = samples_df.PureCN_comment.str.extract(r'([0-9]+)',expand = True).fillna(100).astype(int)\n",
+    "samples_df['Non_aberrant'] = samples_df.PureCN_comment.str.contains(\"NON-ABERRANT\").astype(bool)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "635ca32b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "samples_df = samples_df[samples_df.PureCN_ploidy != 'NA']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9c072570",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "to_curate = samples_df[((samples_df.PureCN_gof < MIN_GOF) & ~samples_df.Non_aberrant) | \\\n",
+    "                       (samples_df.PureCN_ploidy.astype(float) > MAX_PLOIDY)]\n",
+    "to_curate[\"PureCN_curated_solution\"] = \"\"\n",
+    "to_curate[\"PureCN_failed\"] = \"\"\n",
+    "to_curate[\"PureCN_curated\"] = True\n",
+    "to_curate_df = to_curate.loc[:,['PureCN_ploidy','PureCN_comment','PureCN_curated', \\\n",
+    "           'PureCN_curated_solution','PureCN_failed']]\n",
+    "# Saves lines that need to be manually curated as a csv file\n",
+    "to_curate_df.to_csv(\"to_curate.csv\") "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1a5163a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "to_curate_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88cb0c2a",
+   "metadata": {},
+   "source": [
+    "## Download Solution PDFs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6ee3126",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.system(\"gsutil -m cp \"+to_curate.PureCN_solutions_pdf.str.cat(sep = \" \")+ \" \" + SAVE_PDF_DIR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88252357",
+   "metadata": {},
+   "source": [
+    "# Manually inspect and update solutions "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50baf673",
+   "metadata": {},
+   "source": [
+    "Now that all the solution PDFs and \"to_curate.csv\" are downloaded, open the solution PDFs locally and manually inspect them following the guidelines here: https://docs.google.com/document/d/1Rte0xKK3ZE_UV6MWepdXRIbAehUJg8FuLaDckrWhPTQ/edit?usp=sharing. Fill in the PureCN_curated_solution column for all samples in to_curate.csv with the index of the most reasonable solution. Note that solution indices are 1-based. If none of the solutions look right, set PureCN_failed = \"TRUE\" & PureCN_curated_solution = 0."
```

Comment:
list in readme too.

Reply:
I think the readme is a bigger problem (a lot more things are outdated besides mentions of PureCN) so we might want to update it as a whole after this merge?

Reply:
Turns out William had already linked it in the purecn readme!